src/processors/AsyncAPIInputProcessor.ts

```diff
@@ -145,14 +145,30 @@ export class AsyncAPIInputProcessor extends AbstractInputProcessor {
 	 * @param input 
 	 */
   shouldProcess(input: Record<string, any>) : boolean {
-    //Check if we got a parsed document from out parser
-    //Check if we just got provided a pure object
-    if (AsyncAPIInputProcessor.isFromParser(input) || input.asyncapi !== undefined) {
+    const version = this.tryGetVersionOfDocument(input);
+    if (version === undefined) { return false; }
+    const supportedVersions = ['2.0.0', '2.1.0'];
+    if (supportedVersions.includes(version)) {
       return true;
     }
     return false;
```

Comment:
```suggestion
    const version = this.tryGetVersionOfDocument(input);
    return supportedVersions.includes(version);
```

and 

```js
const supportedVersions = ['2.0.0', '2.1.0'];
```

should be treated as `private static` value for class or `const` outside the class :)

Reply:
Good call 👍